### DISCUSSION TITLE
intellisense does not work

### DISCUSSION THREAD

**Discussion by josegduarte96** [quasarframework/quasar#16362] [General - Components / Directives / etc] [answered]
Started a new project with quasar CLI (@quasar/cli v2.3.0) and it does not offer component autocompletion or suggest props. Someone else happens?

![image](https://github.com/quasarframework/quasar/assets/67060283/534e72da-b702-43ae-a98b-ae113256ff7b)

**Comment by yusufkandemir** [comment, marked as answer]
Restart VS Code (_or just Volar_) and it should work fine. Volar doesn't watch `node_modules`, so you have to restart it manually when you install/update packages.

**Comment by focus1921** [comment]
Quoted from my [post](https://jzwangblog.wordpress.com/2024/02/18/how-to-resolve-the-issue-of-quasar-v2-project-lacking-intellisense-in-vs-code/):

>Open the Quasar project and create a new file named 'quasar.d.ts' in the 'src' directory.
>
>If you are working with 'Quasar CLI with Vite', add the following content into the 'quasar.d.ts' file.
>
>`/// <reference types="@quasar/app-vite" />`
>
>If you are working with 'Quasar CLI with Webpack', add the following content into the 'quasar.d.ts' file.
>
>`/// <reference types="@quasar/app-webpack" />`
>
>Restart VS Code and the Intellisense should work fine.
